staging_service/import_specifications/individual_parsers.py

```diff
@@ -351,15 +357,26 @@ def parse_dts_manifest(path: Path, validator: Draft202012Validator) -> ParseResu
     results = {}
     try:
         with open(path, "r") as manifest:
-            manifest_json = json.load(manifest)
-        if not isinstance(manifest_json, dict):
+            manifest = json.load(manifest)
+        if not isinstance(manifest, dict):
             return _error(Error(ErrorType.PARSE_FAIL, "Manifest is not a dictionary", spcsrc))
-        for err in validator.iter_errors(manifest_json):
+        for err in validator.iter_errors(manifest):
             err_str = err.message
             err_path = list(err.absolute_path)
             if err_path:
                 err_str += f" at {err_path}"
             errors.append(Error(ErrorType.PARSE_FAIL, err_str, spcsrc))
+        if not errors:
+            results = defaultdict(list)
+            instructions = manifest[_DTS_INSTRUCTIONS_KEY]
+            for resource_obj in instructions[_DTS_INSTRUCTIONS_OBJECTS_KEY]:
+                datatype = resource_obj[_DTS_INSTRUCTIONS_DATATYPE_KEY]
+                parameters = frozendict(resource_obj[_DTS_INSTRUCTIONS_PARAMETERS_KEY])
+                results[datatype].append(parameters)
+            # Re-package results as a dict of {datatype: ParseResult}
+            results = {
+                source: ParseResult(spcsrc, tuple(parsed)) for source, parsed in results.items()
```

Comment:
```suggestion
                datatype: ParseResult(spcsrc, tuple(paramlist)) for datatype, paramlist in results.items()
```

Reply:
you didn't like the paramlist name? That matched up better with the prior var names to me

Reply:
Oops. Meant to change that and got distracted.